linera-base/src/crypto.rs

```diff
@@ -419,12 +426,14 @@ impl CryptoHash {
         CryptoHash(hasher.finalize())
     }
 
+    /// Reads the bytes of the hash value.
```

Comment:
```suggestion
    /// Returns a reference to the bytes of the hash value.
```
(Just to clarify that it doesn't actively copy or read.)